packages/client/src/shared/services/http.js

```diff
@@ -0,0 +1,60 @@
+export async function postRequest (url, payload) {
```

Comment:
Curious, could we not just make 1 function? and just pass in the `method` and payload is optional?
They all seem the same? 

Though im not really bothered - just a thought ^_^

Reply:
Yep I like!